### PR TITLE
Move to PackageReference and 4.7.2.

### PR DESCRIPTION
**Pull request by KirillOsenkov**

### CODE REVIEW

File: Mono.Debugger.Soft/Mono.Debugger.Soft.csproj [hunk 1]
```diff
@@ -118,22 +118,12 @@
   <ItemGroup>
     <None Include="Makefile.am" />
     <None Include="mono-git-revision" />
-    <None Include="packages.config" />
+  </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="Mono.Cecil" Version="0.10.1" PrivateAssets="all" />
```
Comment (sandyarmstrong) [inline Mono.Debugger.Soft/Mono.Debugger.Soft.csproj]: Anyway way to get these using `$(NuGetVersionCecil)` when built in MonoDevelop? Is that worthwhile or no? I'm not sure how many ways these are built/distributed.

Comment (KirillOsenkov) [reply Mono.Debugger.Soft/Mono.Debugger.Soft.csproj]: I've added PrivateAssets="all" so it doesn't get inherited by referencing projects.

Comment (Therzok) [reply Mono.Debugger.Soft/Mono.Debugger.Soft.csproj]: @KirillOsenkov can we use the same patetrn as in RefactoringEssentials?

https://github.com/mono/RefactoringEssentials/blob/monodevelop2/RefactoringEssentials/RefactoringEssentials.csproj#L24-L26

Comment (KirillOsenkov) [reply Mono.Debugger.Soft/Mono.Debugger.Soft.csproj]: good idea, will do.

**Pull request merged**